cbmc/proofs/Makefile.common

```diff
@@ -75,17 +75,18 @@ CBMC_MAX_OBJECT_SIZE = "(SIZE_MAX>>(CBMC_OBJECT_BITS+1))"
 CBMCFLAGS += \
 	$(UNWINDING) \
 	--bounds-check \
-	--pointer-check \
-	--unwinding-assertions \
-	--nondet-static \
+	--conversion-check \
 	--div-by-zero-check \
 	--float-overflow-check \
+	--flush \
 	--nan-check \
+	--nondet-static \
+	--pointer-check \
 	--pointer-overflow-check \
-	--undefined-shift-check \
 	--signed-overflow-check \
+	--undefined-shift-check \
 	--unsigned-overflow-check \
-	--flush \
+	--unwinding-assertions \
```

Comment:
Can you confirm this is doing nothing but adding conversion-check and sorting?

Reply:
Yes, it only adding conversion-check and sorting.